test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/MvcTagHelpersWebSite.MvcTagHelper_Home.Order.html

```diff
@@ -19,7 +19,15 @@
         <div>
             <label class="order" for="Products">Products</label>
 
-            <select multiple="multiple" id="Products" name="Products"><option value="0">Product_0</option>
+            <select multiple="multiple" id="Products" name="Products"><option selected="selected" value="0">Product_0</option>
+<option selected="selected" value="1">Product_1</option>
+<option value="2">Product_2</option>
+</select>
+        </div>
+        <div>
+            <label class="order" for="SubstituteProducts">SubstituteProducts</label>
+            
+            <select id="SubstituteProducts" multiple="multiple" name="SubstituteProducts"><option value="0">Product_0</option>
 <option value="1">Product_1</option>
```

Comment:
only intentional if you call anything Razor generates intentional :smiling_imp:  not something we can fix here